ext/liquid_c/vm.c

```diff
@@ -313,13 +322,18 @@ static VALUE vm_render_until_error(VALUE uncast_args)
             case OP_BUILTIN_FILTER:
             {
                 VALUE filter_name;
+                uint8_t num_args;
+
                 if (ip[-1] == OP_FILTER) {
-                    filter_name = *const_ptr++;
+                    filter_name = RARRAY_AREF(constant, 0);
+                    num_args = RARRAY_AREF(constant, 1);
```

Comment:
Calling the same filter with a different number of arguements should be fine...

I had to store the `num_args` in the `constants_table` since I couldn't fit it in the instruction...
I had to come up with a cheeky solution where I store the `filter_name` and `num_args` together in the `constants_table`:
Before:
```
Insturction: [OP_FILTER][num_args] # 8 + 8 bit
Constants:   [filter_name] # 8 bit
```

After:
```
Insturction: [OP_FILTER][constant_index] # 8 + 16 bit (24 bit limit)
Constants: [[filter_name, num_args]]
```

Reply:
Aaaah got it! I had the wrong assumption that filter names were reused in the constant table 👍